# kaleido cache provider messages
cacheprovider.default=Setting "kaleido.cacheprovider" to "{0}" 
cacheprovider.customize=You can set the default cache provider, by defining the java system variable as "-Dkaleido.cacheprovider=<local|ehCache|gae|infinispan|coherence3x|websphere|jbossCache3x>" 
cacheprovider.notfound=None cache provider found for "{0}". A local java.util.concurrent.ConcurrentHashMap will be used. Please check java system variable:  "-Dkaleido.cacheprovider=<code>"  , using one of these code "local|ehCache|gae|infinispan|coherence3x|websphere|jbossCache3x"
cacheprovider.illegal=None cache provider found for "{0}". Default provider codes are: "local|ehCache|gae|infinispan|coherence3x|websphere|jbossCache3x"
cacheprovider.classcasterror=The plugin name "{0}" is referencing the class type "{1}" instead of "{2}"

cachemanager.loading.default=Creating cache manager "{0}"
cachemanager.loading.custom=Creating cache manager "{0}" using configuration "{1}"
cachemanager.create.cache=Creating cache "{0}"
cachemanager.create.cache.detailed=Creating cache "{0}" using cache manager "{1}"
cachemanager.destroy.info=Destroying cache "{0}"...
cachemanager.destroyall.store.error=Error while unloading the internal configuration resource
cachemanager.destroyall.error=Error while releasing the cache manager resources "{0}"
cachemanager.classloader.notfound=Class loader "{0}" cannot be found. Please check your runtime context configuration

# cache messages
cache.destroy.access=The cache "{0}" was destroyed by another process, you you cannot anymore access it
cache.clear.error=Error while clearing the cache entries
cache.put.error=Error while putting a new entry in the cache

cache.configuration.notfound=Configuration "{0}" cannot be found in the classpath "{1}". Please check traces for more information.
cache.configuration.notCachefound=No configuration entry found for cache name "{0}" in the resource "{1}".
cache.configuration.error=Cache provider "{0}" met a configuration exception. Please have a look to "{1}"
cache.configuration.ioe.error=IO error while loading the cache configuration. Please check traces for more information.


